src/Routes.js

```diff
@@ -11,58 +11,27 @@ import {
   ProfilePage,
   CompanyPage,
   TalkPage,
+  SettingsPage,
 } from "./components/pages";
 //#endregion
 
 //#region > Components
 /** @class Route component which includes all routes to specified components */
 class Routes extends React.Component {
   render() {
-    const { globalState, globalFunctions } = this.props;
-
     return (
       <Switch>
-        <Route
-          exact
-          path="/"
-          component={(props) => (
-            <HomePage
-              globalFunctions={globalFunctions}
-              globalState={globalState}
-              {...props}
-            />
-          )}
-        />
+        <Route exact path="/" component={(props) => <HomePage {...props} />} />
```

Comment:
This is because it should be a kind of template. If props should be handed over at some point, it works. However, it is currently not needed in any case. It is true that it is illogical why it is sometimes missing. Therefore it should be added to every route component.